lib/utils.js

```diff
@@ -1,7 +1,12 @@
 'use strict';
+
 var support = require('./support');
-var compressions = require('./compressions');
-var nodeBuffer = require('./nodeBuffer');
+var nodejsUtils = require('./nodejsUtils');
+
+// "somewhere early in your app; it attaches to the global."
+// Is updating the global scope really a good idea ?
+require("setimmediate2");
```

Comment:
There's also [asap](https://www.npmjs.com/package/asap) which is a simple function, and is probably better than updating a global